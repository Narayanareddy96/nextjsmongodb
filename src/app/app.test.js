import { render, screen } from '@testing-library/react';
import Dashboard from './dashboard/page';
import Home from './page';

test('Check the Dashboard page', () => {
  // Arrange
  render(<Dashboard />);
  expect(screen.getByText('Dashboard')).toBeInTheDocument();
});

test('Check the Home page', () => {
  // Arrange
  render(<Home />);
  //   expect(screen.getByText('Venkat')).toBeInTheDocument();
});
