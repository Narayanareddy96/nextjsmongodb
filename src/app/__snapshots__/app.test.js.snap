// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check the home page 1`] = `
<div>
  <div>
    Dashboard
  </div>
</div>
`;
